tools/hangar/__snapshots__/test_corpus/valid/resource_captures_globals.w_compile_tf-aws.md

```diff
@@ -91,10 +88,8 @@ module.exports = function({ globalBucket, globalStr, globalBool, globalNum, glob
       this.localTopic = localTopic;
     }
     async $inflight_init()  {
-      const __parent_this = this;
```

Comment:
you just saved lots of trees in the amazon forests by not emitting these redundant statements! thank you!